example/usecase/employer_university_flow/pkg/util.go

```diff
@@ -39,35 +39,34 @@ func GenerateDIDPeer() (did.DID, error) {
 	if err != nil {
 		return nil, err
 	}
-	did, err := did.PeerMethod0{}.Generate(kt, pubKey)
+	dID, err := did.PeerMethod0{}.Generate(kt, pubKey)
 	if err != nil {
 		return nil, err
 	}
-	return did, nil
+	return dID, nil
 }
 
 // This validates the VC.
 // TODO: Expand on this more
 // Simplify it?
 func validateVC(vc credential.VerifiableCredential) error {
-
 	issuer := "https://example.edu/issuers/565049"
-	AssertionMethod := cryptosuite.ProofPurpose("assertionMethod")
+	assertionMethod := cryptosuite.ProofPurpose("assertionMethod")
 
 	var vc2 credential.VerifiableCredential
 	if err := util.Copy(&vc, &vc2); err != nil {
 		return err
 	}
 
-	var OKP = cryptosuite.KTY("OKP")
-	var Ed25519 = cryptosuite.CRV("Ed25519")
+	var okp = cryptosuite.KTY("OKP")
+	var ed25519 = cryptosuite.CRV("Ed25519")
```

Comment:
```suggestion
	okp := cryptosuite.KTY("OKP")
	ed25519 := cryptosuite.CRV("Ed25519")
```

Reply:
actually, I think we already have these types defined. `cryptosuite.OKP` and `cryptosuite.Ed25519` should work